src/app/components/elements/ReplyEditor.scss

```diff
@@ -114,3 +114,7 @@
 .RichTextEditor__root___33zoV button {
   margin-right: 0 !important;
 }
+
+.Dropdown__root___1B9ta {
```

Comment:
is there any way to use a selector that relates to a class we can find in our codebase? it looks like this one is autogenerated during runtime and might change at some point.

if there isn't a way to write a specific rule, we need to modify the dropdown so that it can be more easily styled.